go/controller/cluster.go

```diff
@@ -143,7 +175,9 @@ func (c *Cluster) SyncResource() (res autoscaler.ClusterResource, err error) {
 		GPULimit:        int(allLimits.NvidiaGPU().Value()),
 		CPULimitMilli:   allLimits.Cpu().ScaledValue(resource.Milli),
 		MemoryLimitMega: allLimits.Memory().ScaledValue(resource.Mega),
-	}
 
+		NodesCPUIdleMilli:   nodesCPUIdleMilli,
+		NodesMemoryIdleMega: nodesMemoryIdleMega,
```

Comment:
We say idle cpu and free memory, not idle memory.

Reply:
Done.